assets/src/scripts/components/hydrograph/audible.js

```diff
@@ -74,13 +75,43 @@ export const audibleInterfaceOnSelector = state => state.audibleInterfaceOn;
 
 export const audibleScaleSelector = memoize(tsKey => createSelector(
     yScaleSelector,
-    (scale) => {
+    (yScale) => {
         return scaleLinear()
-            .domain(scale.domain())
+            .domain(yScale.domain())
             .range([80, 1500]);
     }
 ));
 
+const audiblePointsSelector = createSelector(
+    allTimeSeriesSelector,
+    tsCursorPointsSelector('current'),
+    tsCursorPointsSelector('compare'),
+    audibleScaleSelector('current'),
+    audibleScaleSelector('compare'),
+    (allTimeSeries, currentPoints, comparePoints, yScaleCurrent, yScaleCompare) => {
+        // Set null points for all time series, so we can turn audio for those
+        // points off when toggling to other time series.
+        let points = Object.keys(allTimeSeries).reduce((points, tsID) => {
```

Comment:
This is just a general comment. I find using reduce to essentially perform a mapping of values on an object to be confusing. The project already brings in lodash. Thoughts about using _mapValues to do this kind of mapping.

Reply:
I wouldn't be against that.